lms/static/sass/features/_learner-profile.scss

```diff
@@ -164,53 +168,93 @@
 
     .wrapper-profile-sections {
         @extend .container;
-        padding: 0 ($baseline*1.5);
+        @include padding($baseline*1.5, $baseline*1.5, $baseline*1.5, 0);
+        min-width: 0;
+
+        @media (max-width: $learner-profile-container-flex) { // Switch to map-get($grid-breakpoints,md) for bootstrap
+            @include margin-left(0);
+            @include padding($baseline*1.5, 0, $baseline*1.5, 0);
+        }
+    }
+
+    .profile-header {
+        @include padding(0, 0, $baseline, $baseline*3);
+
+        .header {
+            @extend %t-title4;
+            @extend %t-ultrastrong;
+            display: inline-block;
+        }
+
+        .subheader {
+            @extend %t-title6;
+        }
     }
 
     .wrapper-profile-section-one {
-        width: 100%;
-        display: inline-block;
-        margin-top: ($baseline*1.5);
-        @include margin-left($baseline/2);
+        @include float(left);
+        @include margin-left($baseline*3);
+        width: 300px;
+        background-color: $white;
+        border-top: 5px solid $blue;
```

Comment:
Unfortunately the openedx bootstrap variables file only has white set for $body-bg, $btn-primary-color, and input-bg - none of which seemed to make sense for this case. It seems like something that we should add eventually once we start converting to bootstrap (a lot of files are hard coding $white for now), but for now I'll just keep as is.